test/expression/string.stmt

```diff
@@ -0,0 +1,88 @@
+40 columns                              |
+>>> Long single-line.
+"this string is longer than forty characters";
+<<<
+"this string is longer than forty characters";
+>>> Empty multi-line.
+{""""""; '''''';}
+<<<
+{
+  """""";
+  '''''';
+}
+>>> Blank lines in multi-line.
+'''
+
+
+two before
+
+one
+
+
+two
+
+
+''';
+<<<
+'''
+
+
+two before
+
+one
+
+
+two
+
+
+''';
+>>> Short one line multi-line.
+"""not too long""";
+<<<
+"""not too long""";
+>>> Multi-line with short lines.
+"""
+not too long
+or this one
+""";
+<<<
+"""
+not too long
+or this one
+""";
+>>> Multi-line with long lines.
+"""
+this string is longer than forty characters
+this one is also is longer than forty characters
+""";
+<<<
+"""
+this string is longer than forty characters
+this one is also is longer than forty characters
+""";
+>>> Only indent the first line of multiline strings.
+{
+"""
+multiline
+""";
+}
+<<<
+{
+  """
+multiline
+""";
+}
+>>>
```

Comment:
Nit: Write a small description for this test?

Reply:
Done. (I tended to omit the description if a test case is for the same reason as the preceding one, but I just realized I never once explained that convention and maybe it's not worth keeping. :) )